src/common.jl

```diff
@@ -75,7 +75,11 @@ function _get_withdrawals!(
     bus_lookup::Dict{Int, Int},
     sys::PSY.System,
 )
-    loads = PSY.get_components(x -> !isa(x, PSY.FixedAdmittance), PSY.ElectricLoad, sys)
+    loads = PSY.get_components(
```

Comment:
In this case it depends on the control mode of the switched admittance. We need to get this filters in a separate function probably and have more complex queries.